### PR TITLE
Teach Trog some grammar

### PR DESCRIPTION
**Pull request by RojjaCebolla**
example item names now possible:

the Foo...
(general)
of Hungering Death
of the Rescuing Gods
of the Glowing God
of Uskayaw, Screaming
(trog)
of Glamorous Victory
of the Hopping Regicide
of Barnacled Death
(oka)
of Okawaru, Healing
(xom)
"Xom, Gnashing and Baking"
of Healing Xom

is the comment still true that item names are limited to 25 characters? i wasn't able to find that in the source. If it is, "Xom, Gnashing and Baking" would just barely be rejected.

edit:  "Xom Gnashing 'n Baking" would fit, though

### CODE REVIEW

**Comment by RojjaCebolla** [comment]
"Xom Gnashing 'n Baking" would fit though

File: crawl-ref/source/dat/database/randname.txt [hunk 1]
```diff
@@ -73,9 +73,11 @@ of @_battle_or_war_@
 w:1
 of @player_doom@
 
-w:9
+w:8
 of @death_or_doom@
 
+ w:1 of @_verbing_@ @death_or_doom@
```
Comment (ebering) [inline crawl-ref/source/dat/database/randname.txt:79]:
```suggestion
w:1
of @_verbing_@ @death_or_doom@
```

File: crawl-ref/source/dat/database/randname.txt [hunk 2]
```diff
@@ -700,7 +750,7 @@ w:5
 "Gaze Upon Infinity"
 
 w:5
-"I Knew You'd Do That"
+"I Foresaw This"
```
Comment (ebering) [inline crawl-ref/source/dat/database/randname.txt:753]: Why the change to this one, instead of both?

File: crawl-ref/source/dat/database/randname.txt [hunk 3]
```diff
@@ -1118,7 +1168,7 @@ jewelled
 
 ichor-stained
 
-distressingly furry
+surprisingly fuzzy
```
Comment (ebering) [inline crawl-ref/source/dat/database/randname.txt:1171]:
```suggestion
distressingly furry
```

This one is a crawl classic!!!

File: crawl-ref/source/dat/database/randname.txt [hunk 4]
```diff
@@ -1082,7 +1132,7 @@ heavily jewelled
...
```
Comment (ebering) [inline crawl-ref/source/dat/database/randname.txt:1135]:
```suggestion
distressingly furry
```

**Review by ebering — CHANGES_REQUESTED** [review]
Thanks for the db additions! But the removals run from "ok?" to "how dare you remove a crawl classic name without comment?!?!?!", I'll make the changes either in the web ui or command line and merge.

File: crawl-ref/source/dat/database/randname.txt [hunk 5]
```diff
@@ -700,7 +750,7 @@ w:5
 "Gaze Upon Infinity"
 
 w:5
```
Comment (ebering) [inline crawl-ref/source/dat/database/randname.txt:752]:
```suggestion
w:5
"I Knew You'd Do That"

w:5
```

**Review by ebering — APPROVED** [review]

File: crawl-ref/source/dat/database/randname.txt (hunk 4 above)
Comment (RojjaCebolla) [reply crawl-ref/source/dat/database/randname.txt]: thanks for the review! when i've introducing new players to crawl, two or three different people have said something along the lines of "what's this? random subtle furry hate in this game?"

File: crawl-ref/source/dat/database/randname.txt (hunk 2 above)
Comment (RojjaCebolla) [reply crawl-ref/source/dat/database/randname.txt]: i didn't see there was another one. this one just felt a little odd and ... not *deific* enough for an Ash name. but regular randarts get to be very silly, so maybe it's fine for a few ash artis to border on silliness too?

File: crawl-ref/source/dat/database/randname.txt (hunk 3 above)
Comment (RojjaCebolla) [reply crawl-ref/source/dat/database/randname.txt]: it is! but when i've introduced new players to crawl, two or three different people have said something along the lines of "what's this? random subtle furry hate in this game?"
